glide.yaml

```diff
@@ -0,0 +1,20 @@
+package: github.com/gametimesf/kinesis-connectors
```

Comment:
Why are you changing the build system of someone else's software? This can never get merged back.